examples/pygstc/simple_pipeline.py

```diff
@@ -0,0 +1,167 @@
+#The software contained in this file is free and unencumbered software released into the public domain.
+#Anyone is free to use the software contained in this file as they choose, including incorporating
+#it into proprietary software.
+import time
+import sys
+import threading
+from pygstc.gstc import GstcError, GstdError, GstdClient
+from pygstc.logger import CustomLogger
+
+class GstcPlayer:
+
+  def __init__(self):
+    #Create a custom logger with loglevel=DEBUG
+    self.gstd_logger = CustomLogger('simple_playback', loglevel='DEBUG')
+    #Create the client with the logger
+    self.gstc = GstdClient(logger=self.gstd_logger)
+    self.pipeline = "fakesrc ! fakesink"
+
+    #Check if pipe is already created
+    if ( self.pipe_exists("p0")):
+      self.gstc.pipeline_delete("p0")
+    self.pipe_created = False
+
+    #Error handler thread
+    self.running = True
+    self.thErrorHandler = threading.Thread(target=self.errPlayerHandler, args=())
+    self.thErrorHandler.daemon = True
+    self.thErrorHandler.start()
+
+  def errPlayerHandler(self):
+    while (self.running):
+      if(self.pipe_created):
+        self.gstc.bus_timeout("p0", -1)
+        self.gstc.bus_filter("p0", "error+eos+warning")
+        resp = self.gstc.bus_read("p0")
+        print(resp)
+
+  def finish(self):
+    self.running = False
+
+  def openFile(self, videoPath):
+    #Create pipeline object
+    self.pipeline = "filesrc location="+videoPath+" ! decodebin ! videoconvert ! \
+          videoscale ! capsfilter name=cf ! autovideosink"
+
+  def playVideo(self):
+    print("Playing")
+    if (self.pipe_exists("p0")):
+      self.gstc.pipeline_delete("p0")
+
+    self.gstc.pipeline_create("p0", self.pipeline)
+    self.pipe_created = True
+    self.gstc.pipeline_play("p0")
+
+  def pauseVideo(self):
+    print("Playing")
+    self.gstc.pipeline_pause("p0")
+
+  def continueVideo(self):
```

Comment:
This is very counterintuitive when using a player. I would expect `play` to resume the video.